doc/rst/developer/bestPractices/git.rst

```diff
@@ -1,3 +1,5 @@
+.. _best-practices-git:
+
```

Comment:
The "final merge message" link in the "How to open a PR" section is broken

Reply:
The same link is also broken in "How to merge a PR", as is the first link in that section (which refers to something different)

Reply:
Changing it to a `:ref:` style link fixed it